conf.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+sys.path.insert(0, os.path.abspath('.'))
+
+# -- General configuration ------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.viewcode',
+]
+
+intersphinx_mapping = {'python': ('https://docs.python.org/3.4', None),
+                       'CircuitPython': ('https://circuitpython.readthedocs.io/en/latest/', None)}
+
+# Libraries we depend on but don't need for generating docs.
+autodoc_mock_imports = []
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'README'
+
+# General information about the project.
+project = u'Adafruit HT16K33 Library'
+copyright = u'2016 Radomir Dopieralski'
+author = u'Radomir Dopieralski'
```

Comment:
I'm not the author of this library, or I must have been very drunk and don't remember it.